Tests/OctopusDSC.Tests.ps1

```diff
@@ -35,44 +39,59 @@ Describe "OctopusDSC.psd1" {
 }
 
 Describe "Mandatory Parameters" {
-    It "All required or key properties in the mof file should be marked with [Parameter(Mandatory)} in the psm1 file" {
-        $schemaMofFiles = Get-ChildItem ./OctopusDSC/DSCResources -Recurse -Filter *.mof
-        foreach ($schemaMofFile in $schemaMofFiles) {
-            $schemaMofFileContent = Get-Content $schemaMofFile.FullName
-            $moduleFile = Get-Item ($schemaMofFile.FullName -replace ".schema.mof", ".psm1")
+    $path = Resolve-Path "$PSCommandPath/../../OctopusDSC/DSCResources"
```

Comment:
Bizarrely, this test wasn't working. I know it did at one point. I've now fixed it and made it do lots of if `it {}` blocks rather than one massive test that tested lots of things.